submit_cycle.sh

```diff
@@ -240,71 +236,88 @@ while [ $date_count -lt $dates_per_job ]; do
 
         n_ens=$((n_ens+1))
 
-    done # n_ens < ensemble_size
```

Comment:
Change this back. We need to complete the current PR before moving to adding a new feature